source/__snapshots__/default-highway-test.snap

```diff
@@ -1552,6 +1644,98 @@ export type RuntimeCustomAttributeConfigurationName =
 export type RuntimeCustomAttributeConfigurationLabel =
   RuntimeCustomAttributeConfiguration["label"];
 
+type CustomAttributeValue_somedynamicenumerator = Omit<
```

Comment:
Would it be possible to use a generic instead of suffixes? I haven't tried, but something like:

```
interface TypedCustomAttributeValueMap {
  somedynamicenumerator: string[];
  someboolean: boolean;
}

type TypedCustomAttributeValue<K extends keyof TypedCustomAttributeValueMap> = Omit<
  ContextCustomAttributeValue,
  "key" | "value"
> & {
  key: K;
  value: ValueMap[K];
};

```

Reply:
That also removes the need for the other comment - having a single ```
Omit<
  ContextCustomAttributeValue,
  "key" | "value"
>
``` feels fine